src/mjolnir/convert_transit.cc

```diff
@@ -100,13 +100,14 @@ struct StopEdges {
   std::vector<TransitLine> lines;                     // Set of unique route/stop pairs
 };
 
-// Get scheduled departures for a stop
+// Get scheduled departures for a stop; here we also look at the .pbf.x files,
+// as there can be only stop pairs in the extended ones
```

Comment:
I did what I didn't want to do: mix variable renaming with functional changes.. I just couldn't hold it, was really illegible and then I didn't want to drop that work either. You can ignore everything except the stuff I'll comment on specifically in this file.